internal/installcfg/installcfg.go

```diff
@@ -105,6 +106,10 @@ func getBMHName(host *models.Host, masterIdx, workerIdx *int) string {
 }
 
 func getBasicInstallConfig(cluster *common.Cluster) *InstallerConfigBaremetal {
+	networkType := "OpenShiftSDN"
```

Comment:
maybe it should be replaced with a helper function? something like `getNetworkType`? this function is long.

Reply:
OK